src/StaticWebAssetsSdk/Tasks/WriteImportMapToHtml.cs

```diff
@@ -42,9 +42,11 @@ public partial class WriteImportMapToHtml : Task
     // 2.group = fingerprint placeholder
     // 3.group = file extension
     // wrapped in quotes
-    private static readonly Regex _assetsRegex = new Regex(@"""([^""]+)(#\[\.{fingerprint}\])([^""]+)""");
+    internal static readonly Regex _assetsRegex = new Regex(@"""([^""]+)(#\[\.{fingerprint}\])([^""]+)""");
+
+    internal static readonly Regex _importMapRegex = new Regex(@"<script\s+type=""importmap""\s*>\s*</script>");
 
-    private static readonly Regex _importMapRegex = new Regex(@"<script\s+type=""importmap""\s*>\s*</script>");
+    internal static readonly Regex _preloadRegex = new Regex(@"<link\s+rel=""preload""(\sid=""(?<group>[^""]+)"")?\s*[/]?>");
```

Comment:
add comments about groups same as above

Reply:
I have used named groups in the other regex instead